packages/core/src/provider/initConfigs.ts

```diff
@@ -54,6 +86,12 @@ export async function initRsbuildConfig({
   context.normalizedConfig = normalizeConfig(context.config);
   updateContextByNormalizedConfig(context, context.normalizedConfig);
 
+  const environments = getEnvironmentsConfig(context.normalizedConfig);
+
+  context.environmentConfigs = environments;
```

Comment:
Can we just use `context.normalizedConfig.environments.[name]`?

So we do not need to add a new `context.environmentConfigs` property.

Reply:
done